src/client/sandbox/event/simulator.js

```diff
@@ -610,6 +646,10 @@ export default class EventSimulator {
         return this._dispatchEvent(el, 'focus', false, this.DISPATCHED_EVENT_FLAG);
     }
 
+    storage (window, options) {
```

Comment:
I think that storage event should be mark as private event (`_storage`) because it will not be use in user tests.